batch/sql/insert_local_ssd_resource.py

```diff
@@ -0,0 +1,43 @@
+import asyncio
+import os
+
+from gear import Database
+from hailtop.utils import rate_gib_month_to_mib_msec
+
+
+async def main():
+    # https://cloud.google.com/compute/disks-image-pricing#localssdpricing
+    # https://cloud.google.com/compute/disks-image-pricing#persistentdisk
+    rates = [
+        ('disk/local-ssd/1', rate_gib_month_to_mib_msec(0.048)),
+        ('disk/pd-ssd/1', rate_gib_month_to_mib_msec(0.17))
```

Comment:
Just confirming: you're certain we're using zonal persistent ssds, *not* regional persistent ssds?

Reply:
Yes. They can't be boot disks. https://cloud.google.com/compute/docs/disks/regional-persistent-disk